website/docs/r/rds_instance_v3.html.markdown

```diff
@@ -0,0 +1,259 @@
+---
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_rds_instance_v3"
+sidebar_current: "docs-opentelekomcloud-resource-rds-instance-v3"
+description: |-
+  instance management
+---
+
+# opentelekomcloud\_rds\_instance\_v3
+
+instance management
+
+## Example Usage
+
+### create a single db instance
+
+```hcl
+resource "opentelekomcloud_networking_secgroup_v2" "secgroup" {
+  name = "terraform_test_security_group"
+  description = "terraform security group acceptance test"
+}
+
+resource "opentelekomcloud_rds_instance_v3" "instance" {
+  availability_zone = "{{ availability_zone }}"
+  db = {
+    password = "Huangwei!120521"
+    type = "PostgreSQL"
+    version = "9.5"
+    port = "8635"
+  }
+  name = "terraform_test_rds_instance"
+  security_group_id = "${opentelekomcloud_networking_secgroup_v2.secgroup.id}"
+  network_id = "{{ network_id }}"
+  vpc_id = "{{ vpc_id }}"
+  volume {
+    type = "COMMON"
+    size = 100
+  }
+  flavor = "rds.pg.c2.medium"
+  backup_strategy = {
+    start_time = "08:00-09:00"
+    keep_days = 1
+  }
+}
+```
+
+### create a primary/standby db instance
+
+```hcl
+resource "opentelekomcloud_networking_secgroup_v2" "secgroup" {
+  name = "terraform_test_security_group"
+  description = "terraform security group acceptance test"
+}
+
+resource "opentelekomcloud_rds_instance_v3" "instance" {
+  availability_zone = "{{ availability_zone_1 }},{{ availability_zone_2 }}"
+  db = {
+    password = "Huangwei!120521"
+    type = "PostgreSQL"
+    version = "9.5"
+    port = "8635"
+  }
+  name = "terraform_test_rds_instance"
+  security_group_id = "${opentelekomcloud_networking_secgroup_v2.secgroup.id}"
+  network_id = "{{ network_id }}"
+  vpc_id = "{{ vpc_id }}" 
+  volume {
+    type = "COMMON"
+    size = 100
+  }
+  flavor = "rds.pg.s1.medium.ha"
+  ha_replication_mode = "async"
+  backup_strategy = {
+    start_time = "08:00-09:00"
+    keep_days = 1
+  }
+}
+```
```

Comment:
Can you please add a example where the disk encryption is used?

Am I right that I can use `opentelekomcloud_kms_key_v1` therefor?

```hcl
resource "opentelekomcloud_kms_key_v1" "key_1" {
  key_alias       = "key_1"
  realm           = "eu-de"
}

resource "opentelekomcloud_rds_instance_v3" "instance" {
 ...
  volume {
    disk_encryption_id = "${opentelekomcloud_kms_key_v1.key_1.id}"
    type = "COMMON"
    size = 100
  }
 ...
}
```

Reply:
you are correct.

Reply:
Perfect and thank you for adding the section in the documentation!